pkg/occlient/occlient.go

```diff
@@ -3318,3 +3238,48 @@ func (c *Client) StartDeployment(deploymentName string) (string, error) {
 
 	return result.Name, nil
 }
+
+func uniqueUpsertEnvVarsFromS2IPaths(existingVars []corev1.EnvVar, s2iPaths S2IPaths) []corev1.EnvVar {
```

Comment:
can you please add unit test  for this also ?

Reply:
done